cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -988,8 +981,8 @@ def test_monitoring_endpoint_validation(self):
                     check_port_val, mon_port))
             print("MonIP: {}, MonPort: {}".format(mon_ip, mon_port))
 
-        invalid_ip_ports = [("10.22.31.131.43", "5334"), ("", "9194"),
-                            ("10.177.3.81", "90320"), ("", "73422"), ("10.232.12.8", "9092")]
+        invalid_ip_ports = [("10.22.31.131.43", "5334"), ("", "91943"),
```

Comment:
Where did ports 91943 and 90922 come from?

Reply:
These are just unit tests to validate the port is in the correct range.